db/upgrade/odse/routines/006-sp_investigation_event.sql

```diff
@@ -30,7 +30,6 @@ BEGIN
         /*Complete Investigation section*/
         SELECT results.public_health_case_uid,
                results.program_jurisdiction_oid as program_jurisdiction_oid,
-               jc.code                          as jurisdiction_code,
```

Comment:
Does this have any downstream usage in post-processing?

Reply:
did not find any usage in the post-processing procs.